pkg/git/git.go

```diff
@@ -97,6 +120,21 @@ func (git Client) DeleteLocalRepo() error {
 	return fmt.Errorf("no cloned directory to delete")
 }
 
+// RepoName returns the name of the repository given its URL
+func RepoName(repoURL string) (string, error) {
+	// FIXME: urls like git@github.com:weaveworks/eksctl produce the error "first path segment in URL cannot contain colon"
+	u, err := url.Parse(strings.ReplaceAll(repoURL, ":", "/"))
+	if err != nil {
+		return "", errors.Wrapf(err, "unable to parse git url '%s'", repoURL)
```

Comment:
Über nit: `url` -> `URL`, as other messages to the end-user have the uppercased version, e.g.: `URL for the quickstart base repository`.